payloads/v8/user.ts

```diff
@@ -86,19 +86,19 @@ export interface APIUser {
  */
 export const enum UserFlags {
 	None = 0,
-	DiscordEmployee = 1 << 0,
-	PartneredServerOwner = 1 << 1,
-	DiscordHypeSquadEvents = 1 << 2,
+	Staff = 1 << 0,
+	Partner = 1 << 1,
+	Hypesquad = 1 << 2,
 	BugHunterLevel1 = 1 << 3,
-	HypeSquadHouseBravery = 1 << 6,
-	HypeSquadHouseBrilliance = 1 << 7,
-	HypeSquadHouseBalance = 1 << 8,
-	EarlySupporter = 1 << 9,
-	TeamUser = 1 << 10,
+	HypesquadOnlineHouse1 = 1 << 6,
+	HypesquadOnlineHouse2 = 1 << 7,
+	HypesquadOnlineHouse3 = 1 << 8,
+	PremiumEarlySupporter = 1 << 9,
+	TeamPseudoUser = 1 << 10,
 	BugHunterLevel2 = 1 << 14,
 	VerifiedBot = 1 << 16,
-	EarlyVerifiedBotDeveloper = 1 << 17,
-	DiscordCertifiedModerator = 1 << 18,
+	VerifiedDeveloper = 1 << 17,
+	CertifiedModerator = 1 << 18,
```

Comment:
I know this is edging the scope a bit, but I think we should now add the description to each flag too...

Reply:
Done